app/config/paths.js

```diff
@@ -6,7 +6,11 @@ const isDev = require('electron-is-dev');
 
 const cfgFile = '.hyper.js';
 const defaultCfgFile = 'config-default.js';
-const homeDir = homedir();
+let homeDir = homedir();
+
+if (process.platform === 'win32') {
+  homeDir = resolve(homeDir, 'AppData\\Roaming\\Hyper');
```

Comment:
With this in mind, https://github.com/zeit/hyper/pull/170 I believe would also fix the issue in this PR, as well as add the support for XDG_CONFIG_HOME, the original config path, etc.